Bio/codonalign/codonseq.py

```diff
@@ -265,30 +239,19 @@ def full_translate(self, codon_table=default_codon_table, stop_symbol="*"):
 
     def ungap(self, gap=None):
```

Comment:
I made the Seq ungap default to "-" since that was the default in the Gapped alphabet class, do the same here?

Reply:
OK